arm/ui/templates/login.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+{% block title %}Logs{% endblock %}
+
+{% block nav %}{{ super() }}{% endblock %}
+
+{% block content %}  
+<div class="container">
+    <div class="row h-100 mx-auto align-items-center">
+
+                <div class="col-sm-4 mx-auto" style="flex-wrap: nowrap">
+                    <form action="" method="post" novalidate>
+                        {{ form.hidden_tag() }}
+                        <div>{{ form.username.label }}:{{ form.username(size=32) }} </div>
+                        <div>{{ form.password.label }}:{{ form.password(size=32) }}</div>
+                        {{ form.submit() }}
+                    </form>
+                </div>
+    </div>
+</div>
+{% endblock %}
```

Comment:
I can't seem to find where the username and password is set.

Reply:
I appologize, these are never set in this PR. I added it for later use. I have since implemented a login page & security , but didnt want to increase the size of this already large PR.

Reply:
Ok, I kept trying to find the location where the cookies and password authent was forced.